api/v1/localvolume_types.go

```diff
@@ -64,6 +64,9 @@ type StorageClassDevice struct {
 	// A list of device paths which would be chosen for local storage.
 	// For example - ["/dev/sda", "/dev/sdb", "/dev/disk/by-id/ata-crucial"]
 	DevicePaths []string `json:"devicePaths,omitempty"`
+	// Boolean flag to enforce wiping existing filesystem signatures from the devices before first use.
```

Comment:
I would add something more explicit, like "This option will destroy all leftover data on the devices before they're used as PersistentVolumes. Use with care."
It should sound dangerous.